packages/react/src/examples/forms/validationStatic.stories.tsx

```diff
@@ -221,6 +228,23 @@ export const Static = () => {
               tooltipText="We will send a confirmation to this email address. You may also receive important updates about your parking permit via email."
             />
           </div>
+          <div className="hds-example-form__grid-6-6">
+            <div className="hds-example-form__item">
+              <PhoneInput
+                id="phoneNumber"
+                name="phoneNumber"
+                label="Phone number"
+                onChange={formik.handleChange}
+                onBlur={formik.handleBlur}
+                pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+                helperText="Use format 123-456-7890"
```

Comment:
Should we use some format that is more familiar for Finnish users?

Reply:
- [ ] Use format that is more familiar to Finnish users